lib/data/services/storage_service/shared_preff_storage_service_impl.dart

```diff
@@ -5,43 +5,49 @@ import 'package:water_tracker/data/models/user_settings.dart';
 import 'package:water_tracker/data/services/storage_service/shared_preff_storage_service.dart';
 
 class SharedPreffStorageImplements extends SharedPreffStorageService {
-  static const userSettingsKey = 'userSettings';
-  static const goalsListKey = 'goalsListKey';
+
+  static const amountOfWaterDrunkKey = 'amountOfWaterDrunk';
```

Comment:
are these keys duplicated in firebase impl? move it to StorageService then to remove duplication